relevanceai/operations_new/ops.py

```diff
@@ -0,0 +1,72 @@
+from typing import Optional
+from relevanceai.dataset.write import Write
+
+
+class Operations(Write):
+    def label(
+        self,
+        vector_field: str,
```

Comment:
should we make this vector_fields, in the case of multivector labelling or just stick with this for now?

Reply:
vector fields

Reply:
@jtwinrelevanceai you said it but forgot to do it I think